src/python/lib/clusterfuzz/stacktraces/constants.py

```diff
@@ -534,6 +534,7 @@
     r'.*libc\+\+\.so',
     r'.*libc\+\+_shared\.so',
     r'.*libstdc\+\+\.so',
+    r'.*libc-\d+\.\d+(?:\.\d+)?\.so',
```

Comment:
Isnt this too specific, how about just
i think libc-.*\.so should be better?

Reply:
I was assuming more conservative on the validation, but it's fine with me either way.

I've updated it per your request.